added spelling-error & grammar-error to text-decoration-line (#38612)
mdn/content: files/en-us/web/css/text-decoration-line/index.md

  - : Each line of text has a decorative line going through its middle.
- `blink`
  - : The text blinks (alternates between visible and invisible). Conforming user agents may not blink the text. This value is **deprecated** in favor of [CSS animations](/en-US/docs/Web/CSS/animation).
- `spelling-error`

>   - : The text decoration use the user agents method of highlighting spelling mistakes, this is a dotted red line in most browsers.

Reviewer: ```suggestion
  - : Each line of text uses the user agents' method of highlighting spelling mistakes, which is a dotted red line in most browsers.
```

- `grammar-error`

>   - : The text decoration use the user agents method of highlighting grammar mistakes, this is a dotted green line in most browsers.

Reviewer: ```suggestion
  - : Each line of text uses the user agents' method of highlighting grammar mistakes, which is a dotted green line in most browsers.
```

> > [!NOTE]
> > When using `spelling-error`, and `grammar-error` values for `text-decoration-line`, the user agent disregards the other sub-properties of {{cssxref("text-decoration")}} (such as {{cssxref("text-underline-position")}}, `color`, or `stroke`).

Reviewer 2: Instead of `color` and `stroke`, do you want to link to `text-decoration-color` and `text-decoration-thickness`.
```suggestion
> [!NOTE]
> When using `spelling-error` and `grammar-error` values, the browser disregards the other properties in the {{cssxref("text-decoration")}} shorthand (such as {{cssxref("text-underline-position")}}, `color`, or `stroke`).
```

}
```

{{EmbedLiveSample('basic_example',,90)}}

### Errors example

> In this example the first paragraph contains a spelling mistake and uses the browsers styling for spelling errors is used on the misspelled word. The second paragraph uses the browsers styling for grammar errors. Browser that do not support these there is no line.

Reviewer 2: ```suggestion
In this example, the first paragraph contains a spelling mistake and uses the browser's styling for spelling errors on the misspelled word. The second paragraph uses the browser's styling for grammar errors. There is no styling change in browsers that do not support these `text-decoration-line` values.
```
